docs/docs/configuration.md

```diff
@@ -306,3 +307,22 @@ Retrieve all paths in a graph with a timeout of 500 milliseconds.
 ```
 GRAPH.QUERY wikipedia "MATCH p=()-[*]->() RETURN p" TIMEOUT 500
 ```
+
+---
+
+### EFFECTS_THRESHOLD
+
+Replicate modification via effect when average modification time > `EFFECTS_THRESHOLD`
+
+#### Default
+
+`EFFECTS_THRESHOLD` is 300 μs.
+
+#### Example
+
+Assume `MATCH (n) WHERE n.id < 100 SET n.v = n.v + 1` updated 5 nodes
+and the query total execution time is 5ms, the average modification time is:
+total execution time / number of changes:  5ms / 5 = 1ms.
+if the average modification time is greater then `EFFECTS_THRESHOLD` the query
+will be replicated to both replicas and AOF as a graph effect otherwise the original
+query will be replicated.
```

Comment:
Suggestion: Add:
0: always replicate the effect; -1:  always replicate the original query.

Reply:
Please add:
(Since RedisGraph v2.12)